### DISCUSSION TITLE
Bleachbit for Flathub?

### DISCUSSION THREAD

**Discussion by A3E3A** [bleachbit/bleachbit#1123] [General]
Hi, would like to know if Bleachbit could be added to Flathub? It would be wonderful to to use in any distro that is currently not supported (Clear Linux for example).

**Comment by az0** [comment]
There is progress on a [Flatpack package](https://github.com/bleachbit/bleachbit/issues/632), but please see this comment about hosting on [Flathub ](https://github.com/bleachbit/bleachbit/issues/342#issuecomment-483932944) or similar service that automatically updates.

**Comment by proletarius101** [comment]
@az0 Are you interested in getting the application verified? https://docs.flathub.org/docs/for-app-authors/verification/
